### PR TITLE
Transforms API: introduce scrub_wikitext and deprecate scrubWikitext

### PR DESCRIPTION
**Pull request by d00rman**
Bug: [T114620](https://phabricator.wikimedia.org/T114620)

### CODE REVIEW

**Comment by d00rman** [comment]
cc @wikimedia/services

File: specs/mediawiki/v1/content.yaml [hunk 1]
```diff
@@ -905,6 +905,16 @@ paths:
           description: The HTML to transform
           type: string
           required: true
+        - name: scrub_wikitext
+          in: formData
+          description: Normalise the DOM to yield cleaner wikitext?
```
Comment (Pchelolo) [inline specs/mediawiki/v1/content.yaml]: Why is there a question mark in the end?

Comment (d00rman) [inline specs/mediawiki/v1/content.yaml]: Because the flag is a boolean, and its value answers that question :)

Comment (Pchelolo) [inline specs/mediawiki/v1/content.yaml]: Ah, now I understand.

**Comment by Pchelolo** [comment]
LGTM :+1:

**Pull request merged**